templates/r10k-code.cronjob.yaml

```diff
@@ -80,7 +80,7 @@ spec:
             - name: r10k-code-secret
               secret:
                 secretName: {{ template "r10k.code.secret" . }}
-                defaultMode: 288 # = mode 0440
+                defaultMode: 292 # = mode 0444
```

Comment:
The `r10k` container runs with the `puppet` username - why is `0444` needed?

Reply:
UPDATE: It runs with that UID but not with the correct GID. A bug has been filed. 
-- https://github.com/puppetlabs/pupperware/issues/194

@underscorgan 
Hey Morgan - tagging you here so that we're all aligned.